en/middleware/micrortps_throughput_test.md

```diff
@@ -93,11 +93,13 @@ while (!_should_exit_task)
 
 Compile and launch both the [client](../middleware/micrortps.md#client-px4-firmware) and the [agent](../middleware/micrortps.md#agent-off-board-fastrtps-interface).
 
+> **Note** You may recall this is a *bidirectional* throughput test, where messages must also be sent from the *Agent* to the *Client*. The good news is that you do not need to modify the Agent code to make this happen. As the *Agent* is an RTPS publisher and subscriber, it will automatically get notified of the RTPS messages it sends, and will then transfer these back to the client.
+
 ## Result
 
-The throughput can be observed in the client shell window.
+The test was executed with PX4 running on Pixracer, connected via a UART to an ordinary PC running Ubuntu 16.04. The default configuration was used for both the Client/Agent configuration.
 
-For PX4 running on Pixracer connected to an ordinary PC running Ubuntu 16.04, the results are:
```

Comment:
For throughput measurement is very important talk about the environment since there are significant variations. I would include a note below with the hardware info.